### PR TITLE
Add Azure Example in README

### PR DESCRIPTION
**Pull request by jwahjudi**
Add example for setting up Sombra in Azure with the Helm chart

Links - https://transcend.height.app/T-35366

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -539,6 +539,104 @@ pathfinder:
...
+  - name: ORGANIZATION_URI
+    value: '<ORGANIZATION_URI>'
+  - name: EMPLOYEE_AUTHENTICATION_METHODS
+    value: 'transcend,session'
+  - name: DATA_SUBJECT_AUTHENTICATION_METHODS
+    value: 'transcend,session'
+  - name: TRANSCEND_URL
+    value: https://api.dev.trancsend.com
+  - name: SOMBRA_ID
+    value: '1af17328-e20c-4ba3-af63-f793cf1ecd13'
```
Comment (shashiranjan84) [inline README.md:601]:
```suggestion
    value: '<SOMBRA_ID>'
```

File: README.md [hunk 2]
```diff
@@ -539,6 +539,104 @@ pathfinder:
...
+        - <SOMBRA_CUSTOMER_INGRESS_DOMAIN>
+
+envs:
+  - name: ORGANIZATION_URI
+    value: '<ORGANIZATION_URI>'
+  - name: EMPLOYEE_AUTHENTICATION_METHODS
+    value: 'transcend,session'
+  - name: DATA_SUBJECT_AUTHENTICATION_METHODS
+    value: 'transcend,session'
+  - name: TRANSCEND_URL
```
Comment (shashiranjan84) [inline README.md:598]: remove this

File: README.md [hunk 3]
```diff
@@ -539,6 +539,104 @@ pathfinder:
...
+    value: https://api.dev.trancsend.com
+  - name: SOMBRA_ID
+    value: '1af17328-e20c-4ba3-af63-f793cf1ecd13'
+  # [Optional] Add the following variables in order to utilize the AWS integrations
+  - name: AWS_ACCESS_KEY_ID
```
Comment (shashiranjan84) [inline README.md:603]: Maybe just update the environment list as this example has nothing to do with AWS integrations

Comment (shashiranjan84) [reply README.md:603]: this https://github.com/transcend-io/sombra-helm-chart/blob/7fe255c247e2a3d32b082e0638dd872a2faf19c4/README.md#configuring-sombra. You can leave the vars here too but also update the example description telling more about AWS integration.

File: README.md [hunk 4]
```diff
@@ -539,6 +539,104 @@ pathfinder:
...
+The following example deploys Sombra on Azure. This would require setting up an Application Gateway, which will be used as the ingress controller, on Azure and adding the TLS secret into the cluster.
+
+```yaml
+imageCredentials:
+  registry: docker.transcend.io
+  username: Transcend
+  password: '<TRANSCEND_API_TOKEN>'
+
+transcend_service:
+  type: NodePort
```
Comment (shashiranjan84) [inline README.md:553]: Also I feel this will work with default `ClusterIP` too

Comment (jwahjudi) [reply README.md:553]: as in we can change the type to ClusterIP?

Comment (shashiranjan84) [reply README.md:553]: yes, basically you remove this `transcend_service` and `customer_service` section and let chart use default one. Please do test it once

File: README.md [hunk 5]
```diff
@@ -606,7 +696,9 @@ The following is a list of enviroment variables supported by Sombra for its conf
...
+| AWS_ACCESS_KEY_ID                      | yes if you're not hosting on AWS and want to utilize any AWS integrations      | N/A  | no     | The AWS access key ID of the IAM user with the STS:AssumeRole                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```
Comment (shashiranjan84) [inline README.md:700]: Even when sombra is hosted in AWS, would this not be required?

Comment (shashiranjan84) [inline README.md:700]:
```suggestion
| AWS_ACCESS_KEY_ID                      | yes if you're not hosting Sombra on AWS and want to utilize any AWS integrations      | N/A  | no     | The AWS access key ID of the IAM user with the STS:AssumeRole                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

File: README.md [hunk 6]
```diff
@@ -606,7 +696,9 @@ The following is a list of enviroment variables supported by Sombra for its conf
...
...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (shashiranjan84) [inline README.md:701]:
```suggestion
| AWS_SECRET_ACCESS_KEY                  |  yes if you're not hosting Sombra on AWS and want to utilize any AWS integrations     | N/A  | no     | The AWS secret access key of the IAM user with the STS:AssumeRole                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

File: README.md (hunk 5 above)
Comment (jwahjudi) [reply README.md]: I think AWS sets this up for you, if you run the instance there?

Comment (jwahjudi) [reply README.md]: I was under the impression thats the case

Comment (shashiranjan84) [reply README.md]: > I think AWS sets this up for you, if you run the instance there?

Can we change it to `yes if you're want to utilize any for the AWS integrations`, which will be valid for any case?

**Review by shashiranjan84 — APPROVED** [review]

**Pull request merged**